Library/ViewModels/MostPopularSearchProjectCellViewModel.swift

```diff
@@ -8,8 +8,10 @@ public protocol MostPopularSearchProjectCellViewModelInputs {
 }
 
 public protocol MostPopularSearchProjectCellViewModelOutputs {
-  var fundingLabelText: Signal<String, NoError> { get }
-  var fundingProgress: Signal<Float, NoError> { get }
+  var deadlineSubtitleLabelText: Signal<String, NoError> { get }
+  var deadlineTitleLabelText: Signal<String, NoError> { get }
+  var fundingSubtitleLabelText: Signal<String, NoError> { get }
+  var fundingTitleLabelText: Signal<String, NoError> { get }
```

Comment:
i see that you've split up the styling of the labels into two parts. that may not always be the best choice with localization (espesh thinking of japan coming soon!), so a more future-proof way would be to use simple html. that requires using our custom `SimpleHTMLLabel` class and having a new string.